qml/DomainGraphLayout.qml

```diff
@@ -1458,4 +1609,87 @@ Item
             }
         }
     }
+
+    // check if model contains entity
+    function contains_entity(domainEntityId, entityId)
+    {
+        // check if domainEntityId has content
+        if (domainEntityId != "")
+        {
+            // belongs to the current domain
+            if (entity_id.toString() != domainEntityId)
+            {
+                return false
+            }
+        }
+        // check all entities by entityId
+
+        // check domain
+        if(entity_id.toString() == entityId)
```

Comment:
This method can be called setting the `domainEntityId`, or not setting it (empty string).

The usage of this method is to ensure the graph gets updated when **ANY** entity alias is renamed (that also applies to Domain). If the dialog has been opened from right-clicking a graph entity reference, `domainEntityId` is known and set. If the dialog has been opened from right-clicking a left-panel entity reference, `domainEntityId` is unknown and set as empty.

This check you are suggesting me to change from `entityId` to `domainEntityId` is only for renaming Domain alias case, so it is correct as it is (as long as it can only be triggered from left-panel and `domainEntityId` would be empty, and the identifier we want to evaluate is the `entityId`.